conf/jex.config

```diff
@@ -0,0 +1,32 @@
+params {
+  config_profile_name = 'Jex'
+  config_profile_description = 'Nextflow config file for the MRC LMS Jex cluster'
+  config_profile_contact = 'George Young (@A-N-Other)'
+  config_profile_url = 'https://lms.mrc.ac.uk/research-facility/bioinformatics-facility/'
+}
+
+process {
+  executor = 'slurm'
+  queue = {
+    if ( task.time <= 6.h && task.cpus <= 8 && task.memory <= 64.GB ) {
+      'nice'
+    } else if ( task.memory > 256.GB ) {
+      'hmem'
+    } else {
+      'cpu'
+    }
+  }
+  clusterOptions = '--qos qos_batch'
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+  cacheDir = '/opt/resources/apps/singularity/cache'
```

Comment:
```suggestion
  libraryDir = '/opt/resources/apps/singularity/cache'
```
`cacheDir` should work as well, but unless everyone can write in that directory, `libraryDir` makes more sense to me.

Reply:
It's a specially mounted area with RW permissions for exactly this purpose. All singularity (not just `nf`-related) images are cached there to avoid the need for repetitive download, so `cacheDir` is fine.